google/cloud/artifactregistry_v1beta2/types/file.py

```diff
@@ -35,7 +32,6 @@
 
 class Hash(proto.Message):
     r"""A hash of file content.
-
```

Comment:
This `\n` disappearing is causing the docs build to fail. I'm not sure why it's only changing here and not for other `proto.Message`s.

Reply:
Filed https://github.com/googleapis/gapic-generator-python/issues/902. I'll add a workaround.